src/utils/Endpoint.js

```diff
@@ -0,0 +1,41 @@
+import transformToHarRequest from './transformToHarRequest';
+import { HTTPSnippet } from 'httpsnippet';
+
+const supportedClients = {
+  curl: {
+    targetId: 'shell',
+    options: { short: true, indent: ' ', prettifyJson: true },
+  },
+};
+
+class Endpoint {
+  constructor(props) {
+    this.path = props.path;
+    this.method = props.method;
+    this.requests = props.requests;
+  }
+
+  static create(data) {
+    const harRequest = transformToHarRequest(data);
```

Comment:
@MeganSteenkamp Suggestion: Remove this module and add the logic into this one instead.

Thoughts:
+ I'm not convinced this module is providing enough value to warrant the extra degree of separation. We're using snapshot tests for this module and we don't need to test the HAR library. Validation could be tested using files
+ We're exposing http snippet details in 2 places (this module and `snippet.convert below`). We should wrap it in a single place. I'm thinking something like `code: createSnippet()` on line 29 instead?

Reply:
@tom-blake I've updated this but I kept the creation of the snippet and the conversion to a string of code separate. The reasoning was that it doesn't make sense to create the same HAR Request repeatedly for each language you want to create a snippet for.